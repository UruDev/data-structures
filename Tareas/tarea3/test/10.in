#es_vacio_binario, insertar_en_binario, imprimir_binario,remover_mayor, remover_de_binario
es_vacio_binario
imprimir_binario
insertar_en_binario (12,m)
insertar_en_binario (8,g)
insertar_en_binario (9,j)
insertar_en_binario (2,c)
insertar_en_binario (1,a)
insertar_en_binario (4,e)
insertar_en_binario (5,h)
insertar_en_binario (7,k)
insertar_en_binario (14,p)
insertar_en_binario (18,s)
insertar_en_binario (1,r)
insertar_en_binario (20,z)
insertar_en_binario (2,n)
es_vacio_binario
imprimir_binario
remover_mayor
remover_de_binario g
imprimir_binario
remover_mayor
remover_de_binario m
imprimir_binario
Fin
