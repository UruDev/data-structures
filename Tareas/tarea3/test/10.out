1>Comando no reconocido.
2>Vacio.
3>
4>Insertado.
5>Insertado.
6>Insertado.
7>Insertado.
8>Insertado.
9>Insertado.
10>Insertado.
11>Insertado.
12>Insertado.
13>Insertado.
14>Insertado.
15>Insertado.
16>Insertado.
17>No vacÃ­o.
18>
---(20,z)
--(18,s)
---(1,r)
-(14,p)
--(2,n)
(12,m)
---(7,k)
--(9,j)
---(5,h)
-(8,g)
---(4,e)
--(2,c)
---(1,a)
19>(20,z)
20>Removido.
21>
--(18,s)
---(1,r)
-(14,p)
--(2,n)
(12,m)
---(7,k)
--(9,j)
---(5,h)
-(4,e)
--(2,c)
---(1,a)
22>(18,s)
23>Removido.
24>
--(1,r)
-(14,p)
--(2,n)
(7,k)
--(9,j)
---(5,h)
-(4,e)
--(2,c)
---(1,a)
25>Fin.
